test/browser/browser.sh

```diff
@@ -22,7 +22,7 @@ dnf install --disablerepo=fedora-cisco-openh264 -y firefox
 sed -ie 's/#SHA_CRYPT_MAX_ROUNDS 5000/SHA_CRYPT_MAX_ROUNDS 5000/' /etc/login.defs
 
 # HACK: https://bugzilla.redhat.com/show_bug.cgi?id=2057769
-if [ "$(rpm -q edk2-ovmf)" = "edk2-ovmf-20220126gitbb1bba3d77-3.el9.noarch" ]; then
+if [ "$(rpm -q edk2-ovmf)" = "edk2-ovmf-20220126gitbb1bba3d77-4.el9.noarch" ]; then
```

Comment:
As the bugzilla got reassigned to libvirt, let's change the version comparison to check for libvirt `-ge 8.1 instead? (I'm not sure exactly which libvirt package)